woocommerce/woocommerce-ios
# Edit Product > Inventory Settings: create a new table view cell for SKU field

SKU field for #1424 

## Summary

This PR created a reusable `UITableViewCell` with a title and text field that is used to show the editable SKU field in Edit Product > Inventory Settings.

## Changes

- Created a reusable cell `TitleAndTextFieldTableViewCell` that contains a title label and an editable text field
- Updated the cell and cell view model for Inventory Settings > SKU to use `TitleAndTextFieldTableViewCell`

## Testing

- Launch the app
- Go to Products tab
- Tap on a Product
- Tap "Edit" in the navigation bar
- Tap the inventory settings row
- Clear the SKU field --> the placeholder state should look as expected
- Enter some text into the SKU field --> the row should look as expected

## Example screenshots

State | Dark | Light
-- | -- | --
Placeholder | ![Simulator Screen Shot - iPhone 11 - 2019-12-24 at 15 50 53](https://user-images.githubusercontent.com/1945542/71402474-f01ada00-2667-11ea-8eea-e9b0ff7ca00b.png) | ![Simulator Screen Shot - iPhone 11 - 2019-12-24 at 16 11 15](https://user-images.githubusercontent.com/1945542/71402544-180a3d80-2668-11ea-955b-a4c8392fc3d1.png)
Non-empty text | ![Simulator Screen Shot - iPhone 11 - 2019-12-24 at 15 51 07](https://user-images.githubusercontent.com/1945542/71402475-f01ada00-2667-11ea-9997-45ee7c9ddc2d.png) | ![Simulator Screen Shot - iPhone 11 - 2019-12-24 at 16 11 20](https://user-images.githubusercontent.com/1945542/71402545-180a3d80-2668-11ea-9255-8603a31238aa.png)




Update release notes:
